editoast/src/main.rs

```diff
@@ -412,6 +422,37 @@ async fn add_electrical_profile_set(
     Ok(())
 }
 
+async fn electrical_profile_set_list(
+    pg_config: PostgresConfig,
+) -> Result<(), Box<dyn Error + Send + Sync>> {
+    let mut conn = PgConnection::establish(&pg_config.url()).await?;
+    let electrical_profile_sets = ElectricalProfileSet::list_light(&mut conn).await.unwrap();
+    println!("Electrical profile sets:\nID - Name");
```

Comment:
I'm always a bit skeptical about commands that print non-parsable output. I think it's a good idea to document the output format though. Maybe add a verbose flag to `Client` and only print extraneous information if `-v` is provided? Wdyt?